src/diffusers/models/unet_2d_condition.py

```diff
@@ -282,6 +284,7 @@ def __init__(
                 dual_cross_attention=dual_cross_attention,
                 use_linear_projection=use_linear_projection,
                 upcast_attention=upcast_attention,
+                ff_multiplier=ff_multiplier,
             )
         elif mid_block_type == "UNetMidBlock2DSimpleCrossAttn":
```

Comment:
Shouldn't we also include it in the case of `UNetMidBlock2DCrossAttn`?

Reply:
The feed forward block is only when a full transformer is used as the attention mechanism. The simple cross attn block just uses a single attention block, so there's no feed forward block

Reply:
ah i see...